nettests/psiphon.go

```diff
@@ -0,0 +1,33 @@
+package nettests
+
+// Psiphon test implementation
+type Psiphon struct {
+}
+
+// Run starts the test
+func (h Psiphon) Run(ctl *Controller) error {
+	builder, err := ctl.Ctx.Session.NewExperimentBuilder(
+		"psiphon",
+	)
+	if err != nil {
+		return err
+	}
+	return ctl.Run(builder, []string{""})
+}
+
+// PsiphonTestKeys contains the test keys
+type PsiphonTestKeys struct {
+	IsAnomaly bool `json:"-"`
```

Comment:
I think we should also expose as a test key:
```
        "bootstrap_time": 6.076582013,
        "failure": null,
```

Reply:
https://github.com/ooni/probe-cli/issues/96

Reply:
See #96